tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Requests/GetOrderItemRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests
+{
+    internal sealed class GetOrderItemRequest
+    {
+        private readonly Request _request;
+        private string _getOrderItemUrl;
```

Comment:
Should be read only?

Reply:
It can't be readonly, due to amending the url in the executeAsync method

Reply:
Oh wait, just seen your comment below. nvm